test/e2e/far_e2e_test.go

```diff
@@ -99,10 +100,29 @@ var _ = Describe("FAR E2e", func() {
 			// save the node's boot time prior to the fence agent call
 			nodeBootTimeBefore, err = e2eUtils.GetBootTime(clientSet, nodeName, testNsName, log)
 			Expect(err).ToNot(HaveOccurred(), "failed to get boot time of the node")
+
+			// set the node as "unhealthy" by disabling kubelet
+			makeNodeUnready(selectedNode)
+
 			far := createFAR(nodeName, fenceAgent, testShareParam, testNodeParam)
 			DeferCleanup(deleteFAR, far)
 		})
+		When("running FAR to reboot one node", func() {
+			It("should successfully remediate it", func() {
+				checkRemediation(nodeName, nodeBootTimeBefore)
+			})
+		})
+
 		When("running FAR to reboot two nodes", func() {
+			AfterEach(func() {
```

Comment:
I don't understand what you're trying to do here...

Can you please put this node selection stuff into the existing BeforeEach(), as suggested earlier already?